### ISSUE TITLE
How does this project relate to libratbag?

### ISSUE THREAD

**Issue by Wolf480pl** [krayon/ratslap#8]
There's another project implementing the protocol for configuring Logitech mice:
https://github.com/libratbag/libratbag

Are these two projects somehow related?
Do you disagree / want to do something differently, or did you just not know about each other?

**Comment by krayon** [comment]
From my understanding they didn't support the G300(s) but do now. They approach things in a much better way probably, as it's designed to be generic whereas I just hacked RatSlap together :P

I do believe they don't quite have the same feature set however, due to their design. IIRC it was to do with setting the DPI levels. Technically neither does RatSlap as I've not committed that branch yet though sanikk has submitted a possible implementation ( see PR #5 ).

I was vaguely aware of libratbag when I wrote this and probably should have written a G300(s) module for them instead however at the time I wasn't planning on releasing this, it was just a quick hack to get it working and "scratch my itch".

In summary, if you use multiple mice, and don't need or want to change DPI levels, I'd probably just use libratbag but if you have a G300(s) and just want a quick tool for it, use RatSlap.
